format/FormatNumber.js

```diff
@@ -306,10 +307,16 @@ function valueColor(v, colorSpec) {
     const defaultColors = {pos: 'xh-green', neg: 'xh-red', neutral: 'xh-gray'};
     colorSpec = typeof colorSpec === 'object' ? colorSpec : defaultColors;
 
-    if (!colorSpec.pos || !colorSpec.neg || !colorSpec.neutral) {
+    if (!colorSpec.pos && !colorSpec.neg && !colorSpec.neutral) {
```

Comment:
Is there value to this now? We are not being very strict about this now, what's the harm in just applying no classes at all if your object is missing these? Also if we do think it's worth throwing we should work on the message:

'Invalid color spec: ' + colorSpec is appearing as 'Invalid color spec: [Object object]' int he console.

Reply:
Agreed - don't think we need it!  I will remove and check in a few other cleanups that came to me while waiting here in the Syracuse airport.  (✈️ deferred from NY so have plenty of time to twiddle these things....)